Fortnox.NET.Tests/Fortnox.NET.Tests/OrderTests.cs

```diff
@@ -94,12 +94,9 @@ public void SearchOrdersTest()
         [TestMethod]
         public void SearchOrdersLastModifiedTest()
         {
-            var dateOfThisTest = DateTime.Parse("2019-10-09 15:25:31");
-            var halfYearPriorToWhenThisTestWasMade = dateOfThisTest.AddDays(-180);
-
             var request = new OrderListRequest(this.connectionSettings.AccessToken, this.connectionSettings.ClientSecret)
             {
-                SearchParameters = new Dictionary<OrderSearchParameters, object> { { OrderSearchParameters.LastModified, halfYearPriorToWhenThisTestWasMade } }
+                SearchParameters = new Dictionary<OrderSearchParameters, object> { { OrderSearchParameters.LastModified, "2019-10-09 15:25:31" } }
```

Comment:
even if the naming might not have been the best, I did like that the variable was named so that it communicated intent or purpose, whereas a magic datetime string doesn't say anything really.

Reply:
The additional variable explains what the date is but doesn't help with explaining the test so I don't think its required here.